integration-cli/docker_cli_swarm_test.go

```diff
@@ -878,12 +878,51 @@ func (s *DockerSwarmSuite) TestDNSConfigUpdate(c *check.C) {
 	c.Assert(strings.TrimSpace(out), checker.Equals, "{[1.2.3.4] [example.com] [timeout:3]}")
 }
 
+// This tries to get the node status, and if it gets "pending", it waits until it's not pending anymore
 func getNodeStatus(c *check.C, d *SwarmDaemon) swarm.LocalNodeState {
-	info, err := d.info()
```

Comment:
An attempt to fix https://jenkins.dockerproject.org/job/Docker%20Master/label=centos72%20&&%20devicemapper/7769/console